src/components/ui/AuthModal/SignUp/validationSchema.ts

```diff
@@ -0,0 +1,28 @@
+import { TFunction } from 'i18next';
+import * as Yup from 'yup';
+
+const getValidationSchema = (t: TFunction<'auth'>) =>
+    Yup.object({
+        fullName: Yup.string()
+            .min(3, t('minNameMsg'))
+            .max(50, t('maxNameMsg'))
+            .required(t('nameRequired')),
+
+        email: Yup.string()
+            .email(t('invalidEmailMsg'))
+            .required(t('emailRequired')),
+
+        phone: Yup.string()
+            .matches(/^[6-9]\d{9}$/, t('invalidPhone'))
+            .required(t('phoneRequired')),
+
+        password: Yup.string()
+            .min(6, t('invalidPhone'))
```

Comment:
I think you have displayed the wrong error message.

Reply:
Fixed it.